swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
+from curses import wrapper
+from re import T
+from signal import set_wakeup_fd
+from pytest import Item
+from swap_meet.item import Item 
+
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory =  None):
+        if inventory == None:
+            inventory = list(())
+        self.item = Item()
```

Comment:
you do not need this line. The Vendor class doesn't need the item class. Item is being passed in when the instance methods are called.